### PR TITLE
Minor edits to Intro to Staking

### LINKED ISSUE #3849: Minor edits Intro to Staking

### PR DESCRIPTION
**Pull request by filippoweb3**
Pretty quick caused a formatting issue at the end of the page, this has been reverted.

### CODE REVIEW

File: docs/learn/learn-staking.md
Comment (alfarok) [inline docs/learn/learn-staking.md]: This will only display properly for Polkadot.  If you visit the same page for Kusama you will get the following formatting:

image

Every time you want to add an RPC component I would think about adding 1 for each network.  Most the time it makes sense because they will each point to different chains, `network="polkadot"` or `network="kusama"`.  In this case, you are pointing at the same chain for both given the context.  It may seem a little redundant but if we omit the conditional rendering brackets prettier will sometimes try and break up the component, causing it to fail.  Prettier ignores trying to format anything inside the `{{ }}`.

This should render properly for both and will not be modified by prettier:

```suggestion
Polkadot currently has
{{ polkadot: <RPC network="polkadot" path="query.staking.validatorCount" defaultValue={297}/> :polkadot }}
{{ kusama: <RPC network="polkadot" path="query.staking.validatorCount" defaultValue={297}/> :kusama }}
validators. The top bound on the number of validators has not been determined yet, but should only
be limited by the bandwidth strain of the network due to peer-to-peer message passing. The estimate
of the number of validators that Polkadot will have at maturity is around 1000. Kusama, Polkadot's
canary network, currently has
{{ polkadot: <RPC network="kusama" path="query.staking.validatorCount" defaultValue={1000}/> :polkadot }}
{{ kusama: <RPC network="kusama" path="query.staking.validatorCount" defaultValue={1000}/> :kusama }}
validator slots in the active set.
```

Let me know if you have any questions! :slightly_smiling_face:

**Review by alfarok — CHANGES_REQUESTED** [review]

File: docs/learn/learn-staking.md
Comment (filippoweb3) [reply docs/learn/learn-staking.md]: I see; thank you! So basically, I use conditional rendering to avoid problems with pretty, even if I do not need it.

**Comment by filippoweb3** [comment]
@alfarok, thank you for the help. Still need approval to merge the PR.

**Review by DrW3RK — APPROVED** [review]

**Comment by DrW3RK** [comment]
Once @alfarok approves, this can be merged

File: docs/learn/learn-staking.md
Comment (alfarok) [reply docs/learn/learn-staking.md]: Exactly

**Review by alfarok — APPROVED** [review]

**Pull request merged**
